metrics/report/report_dockerfile/scaling.R

```diff
@@ -117,6 +119,35 @@ for (currentdir in resultdirs) {
 				skip_points=1
 			}
 
+			# format the pod data from 2 nested columns in a series
+			# of index specific columns to just 2 columns
+			pdata=data.frame(fdata$BootResults$launched_pods)
+			pudata=c()
+			for (i in seq(length(cdata[, "boot_time"]))) {
+				if (i == 1) {
+					# there are no valid values for this index, skipping
+					next
+				}
+				else {
+					# shift to 0 based indexing
+					index=i-1
+					sindex=(index*2)+1
+					eindex=sindex+1
+					row=cbind(pdata[,sindex:eindex])
+					c1=cbind(podname=row[,1])
+					c2=cbind(node=row[,2])
+					c3=cbind(count=rep(i, length(pdata$pod_name)))
+					c4=cbind(boot_time=rep(cdata[, "boot_time"][i],length(pdata$pod_name)))
+					c5=cbind(dataset=rep(testname, length(pdata$pod_name)))
+					prow=cbind(c1,c2,c3,c4,c5)
+					pudata=rbind(pudata,prow)
+				}
+			}
+			# pndata is considered a vector for some reason so converting it to a data.frame
+			pudata=as.data.frame(pudata)
+			pudata$count=as.numeric(as.character(pudata$count))
+			pudata$boot_time=as.numeric(as.character(pudata$boot_time))
+			
```

Comment:
heh, git log shows me a tab on this blank line - we'll mop that up another time.